gen/src/generate/idna/shared/version.rs

```diff
@@ -0,0 +1,55 @@
+// TODO: Unify this with generate::ucd::shared::version
```

Comment:
Seeing the plan to split versions of the different data sources makes it seems like we should keep the generating code split as well. They're different `ReadMe`s anyway, which only happen to have the same version presentation.